dash/frontend/src/app/modules/private/pages/audit-log/audit-log-list/audit-log.component.html

```diff
@@ -2,112 +2,96 @@
-                  </button>
+            <div class="col-xs-12 col-sm-6 col-md-3">
+              <mat-form-field appearance="fill" [formGroup]="filterAuditLogForm">
+                <mat-label>Entity Id</mat-label>
+                <input matInput type="number" formControlName="entityId">
+              </mat-form-field>
+            </div>
+            <div class="col-xs-12" id="auditFilterButtons">
+              <button [disabled]="!filterAuditLogForm.valid"
+                      mat-raised-button id="submitButton"
+                      color="primary"
+                      type="submit" (click)="filterAuditLogs()">Filter
+              </button>
 
-            </form>
+              <button mat-raised-button id="resetButton"
+                      color="primary"
+                      type="submit" (click)="resetAuditLogs()">Reset
+              </button>
+            </div>
           </div>
-          <div class="col-xs"></div>
-        </div>
+        </form>
       </mat-card-content>
     </mat-card>
 
-    <mat-card class="settings-card" *ngIf="showAuditLogTable">
-      <mat-card-header id="auditLogTableHeader">
-        <div class=" page-card-header">
-          <div class="col-xs-9 no-padding">
-            <div class="page-card-title">
-              Audit logs count: {{totalAuditLogs}}
-            </div>
-          </div>
-          <div class="col-xs-3 page-card-title-button-group-right-align">
-            <button class=button [disabled]="totalAuditLogs === 0" (click)="downloadAuditLogs()"
-                    mat-raised-button
-                    color="primary"
-                    type="button">Download
-            </button>
-          </div>
+    <mat-card class="scrollable-table-card" *ngIf="showAuditLogTable">
+      <mat-card-header id="auditLogTableHeader" class="row">
+        <div class="page-card-title col-xs-12 col-sm-9">
+          Audit logs count: {{totalAuditLogs}}
+        </div>
+        <div class="col-xs-12 col-sm-3 start-xs end-sm">
+          <button class=button [disabled]="totalAuditLogs === 0" (click)="downloadAuditLogs()"
+                  mat-raised-button
+                  color="primary"
+                  type="button">Download
+          </button>
         </div>
       </mat-card-header>
-      <div>
-        <mat-card-content>
-          <table mat-table [dataSource]="dataSource" matSort matSortActive="id" matSortDisableClear
+      <mat-card-content>
+        <div class="table-wrapper">
+          <mat-table [dataSource]="dataSource" matSort matSortActive="id" matSortDisableClear
                  matSortDirection="asc">
             <ng-container matColumnDef="id">
-              <th mat-header-cell *matHeaderCellDef mat-sort-header disableClear> Id.</th>
-              <td mat-cell *matCellDef="let auditLog"> {{auditLog?.id}} </td>
+              <mat-header-cell *matHeaderCellDef mat-sort-header disableClear>IS</mat-header-cell>
```

Comment:
I assume this should be ID rather than IS

Reply:
Fixed